vision/unit_tests/test__gax.py

```diff
@@ -23,7 +23,63 @@ def _get_target_class(self):
     def _make_one(self, *args, **kwargs):
         return self._get_target_class()(*args, **kwargs)
 
-    def test_gax_not_implemented(self):
+    def test_ctor(self):
+        from google.cloud.gapic.vision.v1 import image_annotator_client
+
+        client = object()
+        api = self._make_one(client)
+        self.assertEqual(api._client, client)
+        self.assertIsInstance(api._api(),
+                              image_annotator_client.ImageAnnotatorClient)
+
+    def test__to_gapic_feature(self):
+        from google.cloud.vision._gax import _to_gapic_feature
+        from google.cloud.vision.feature import Feature
+        from google.cloud.vision.feature import FeatureTypes
+        from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+        feature = Feature(FeatureTypes.LABEL_DETECTION, 5)
+        feature_pb = _to_gapic_feature(feature)
+        self.assertIsInstance(feature_pb, image_annotator_pb2.Feature)
+        self.assertEqual(feature_pb.type, 4)
+        self.assertEqual(feature_pb.max_results, 5)
+
+    def test__to_gapic_image_content(self):
+        import base64
+        from google.cloud.vision._gax import _to_gapic_image
+        from google.cloud.vision.image import Image
+        from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+        image_content = b'abc 1 2 3'
+        b64_content = base64.b64encode(image_content)
         client = object()
-        with self.assertRaises(NotImplementedError):
-            self._make_one(client=client)
+        image = Image(client, content=image_content)
+        image_pb = _to_gapic_image(image)
+        self.assertIsInstance(image_pb, image_annotator_pb2.Image)
+        self.assertEqual(image_pb.content, b64_content)
+
+    def test__to_gapic_image_uri(self):
+        from google.cloud.vision._gax import _to_gapic_image
+        from google.cloud.vision.image import Image
+        from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+        image_uri = b'gs://1234/34.jpg'
+        client = object()
+        image = Image(client, source_uri=image_uri)
+        image_pb = _to_gapic_image(image)
+        self.assertIsInstance(image_pb, image_annotator_pb2.Image)
+        self.assertEqual(image_pb.source.gcs_image_uri, image_uri)
+
+    def test__to_gapic_image(self):
```

Comment:
- Move this to it's own test class (along with the other 2 test cases; see comment above for the other function)
- The name should describe what's happening, i.e. both source and content are emtpy, so the result is null